File: Java_Backend/E-Tour/src/main/java/dto/SignupRequest.java
package dto;

import java.time.LocalDate;

import lombok.Data;


@Data
public class SignupRequest {
	private String name;
	private String email;
	private String phone;
	private String password;
	private String gender;
	private LocalDate dob;
	private String address;
	private String city;
	private String state;
	private String pincode;

}

File: DotNet_Backend/SafarnamaApplication/SafarnamaApplication/DTOs/SignupRequest.cs
namespace SafarnamaApplication.DTOs
{
    public class SignupRequest
    {
        public string Name { get; set; } = null!;
        public string Email { get; set; } = null!;
        public string Phone { get; set; } = null!;
        public string? Dob { get; set; }    // ISO date string or empty
        public string Gender { get; set; } = "M";
        public string? Address { get; set; }
        public string? City { get; set; }
        public string? State { get; set; }
        public string? Pincode { get; set; }
        public string Password { get; set; } = null!;
    }
}
